vmm_tests/vmm_tests/tests/tests/x86_64.rs

```diff
@@ -143,7 +142,7 @@ async fn vbs_boot_with_tpm(config: PetriVmConfig) -> anyhow::Result<()> {
 }
 
 /// Validate we can reboot a VM and reconnect to pipette.
-// TODO: Reenable interesting guests once #523 is fixed.
+// TODO: Reenable interesting guests once #74 is fixed.
```

Comment:
How does #74 relate to this?

Reply:
It's the underlying issue preventing these tests from passing. It's an openvmm issue, not openhcl, so it hasn't been super high priority.